Makefile

```diff
@@ -196,21 +196,30 @@ endif
 
 PG_REGRESS_ARGS=--no-locale
 
-regresscheck: | install
+# Control installation dependency and wrapper based on NO_INSTALL
+# in the docker environment, we don't need to install the extension
+ifdef NO_INSTALL
```

Comment:
For post-build testing, I need functionality similar to `make installcheck` - specifically to run tests against an existing installation without triggering a new install.

My current approach to modifying the `Makefile` may not be optimal for this use case. Open to suggestions for better solutions.

Reply:
What if we rename `NO_INSTALL` into `IN_DOCKER`/`IS_DOCKER`? It think this might be more universal in future and it might change other variables as well.

Reply:
imho; this functionality is valuable in several non-Docker scenarios; 
like verifying orioledb extensions installed through package managers (apt, yum, etc.)
or running regression tests against production installations.

Therefore, I'd suggest using a more general parameter name that better reflects its purpose.
* `TEST_INSTALLED=1`  ~ we're testing an already installed version
* `SKIP_INSTALL=1`
* `SYSTEM_TEST=1`
* `USE_EXISTING=1`
* `NO_BUILD_INSTALL=1`
* ....